### ISSUE TITLE
create-near-app with angular frontend is broken

### ISSUE THREAD

**Issue by ElderOrb** [near/create-near-app#833]
1. npx create-near-app test --frontend angular
2. cd test
3. yarn dev 

D:\AI\Projects\.fun\near\demo\test>yarn dev                                                                                                                                                                          yarn run v1.22.10
$ nodemon --watch contract -e ts --exec "npm run start"
[nodemon] 2.0.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): contract\**\*
[nodemon] watching extensions: ts
[nodemon] starting `npm run start`

> test@0.0.1 prestart
> npm run build:contract && npm run dev:deploy:contract

> test@0.0.1 build:contract
> node contract/compile.js

> greeter@0.0.1 build
> asb

ln: EPERM: operation not permitted, symlink '../contract/build/release/greeter.wasm' -> 'D:/AI/Projects/.fun/near/demo/test/out/main.wasm'

> test@0.0.1 dev:deploy:contract
> near dev-deploy

Starting deployment. Account id: dev-1616697860647-7195987, node: https://rpc.testnet.near.org, helper: https://helper.testnet.near.org, file: ./out/main.wasm
An error occured
[Error: ENOENT: no such file or directory, open 'D:\AI\Projects\.fun\near\demo\test\out\main.wasm'] {
  [stack]: "Error: ENOENT: no such file or directory, open 'D:\\AI\\Projects\\.fun\\near\\demo\\test\\out\\main.wasm'",
  [message]: "ENOENT: no such file or directory, open 'D:\\AI\\Projects\\.fun\\near\\demo\\test\\out\\main.wasm'",
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'D:\\AI\\Projects\\.fun\\near\\demo\\test\\out\\main.wasm'
}
npm ERR! code 1
npm ERR! path D:\AI\Projects\.fun\near\demo\test
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c near dev-deploy

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ai\AppData\Local\npm-cache\_logs\2021-03-25T18_44_24_252Z-debug.log
npm ERR! code 1
npm ERR! path D:\AI\Projects\.fun\near\demo\test
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c npm run build:contract && npm run dev:deploy:contract

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ai\AppData\Local\npm-cache\_logs\2021-03-25T18_44_24_413Z-debug.log
[nodemon] app crashed - waiting for file changes before starting...

**Comment by chadoh** [comment]
Seems to be a duplicate of #831 and is not Angular-specific, but is related to Windows 10. If this is not accurate, please re-open the issue and let us know!
